packages/rug/src/utils.js

```diff
@@ -26,10 +34,35 @@ export const calcModularScale = curry((scale, exponent) =>
   Math.round(base * Math.pow(scale, Math.round(exponent))) / base
 )
 
+/**
+ * Color Utilities
+ */
+
 export const addUnit = curry((unit, value) => `${value}${unit}`)
 
-export const opacify = (o, rgb) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
+export const opacify = (rgb, o = 1) => rgb.replace('b(', 'ba(').replace(')', `,${o})`)
```

Comment:
At this point I'd strongly prefer to not change the opacity of a color directly at all, ever. `opacify` is really just a helper util for things like shadows.